frontend/src/components/Reports/study/common/ReportByDate.js

```diff
@@ -53,13 +47,29 @@ const ReportByDate = (props) => {
   };
 
   const handleSubmit = () => {
+    if (!reportFormValues.startDate || !reportFormValues.endDate)  {
+      setReportFormValues({
+        ...reportFormValues,
+        error: intl.formatMessage({
+          id: "error.dateRange.start",
+          defaultMessage: "Please select Start and end date.",
+        }),
+      });
+      return;
+    }
+
+    setReportFormValues({
+      ...reportFormValues,
+      error: "",
+    })
+
```

Comment:
Hey @konavivekramakrishna, just wanted to offer a suggestion. You could add a handleChange function that clears any errors. In the video on Loom, I noticed that even if the user selects a date, the error message remains visible

Reply:
Something like `const handleInputChange = (event) => {
    setErrors({});
  };`
aligning with best practices for usability and error handling